cem/test/support-typedef-jsdoc-utils.test.js

```diff
@@ -10,7 +10,7 @@ import {
   findTypePath,
   findInterfacesFromExtends,
   getConstructorNode,
-  getTypesFromConstructor,
+  getTypesFromConstructor, getTypesFromEventTags,
```

Comment:
Can we move this one on its own line? (sorry, I know this should be done by a robot but...)